spec/features/user/dashboard_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'As a user' do
+  before do
+    @mood1 = Mood.create(name: 'worried')
+    @mood2 = Mood.create(name: 'neutral_face')
+    @mood3 = Mood.create(name: 'smile')
+  end
+  it 'once logged in, I can see links to stats, resources and suggested videos' do
+    stub_omniauth
+
+    visit '/login'
+    click_on 'Sign In With Google'
+
+    within('#navbarSupportedContent') do
```

Comment:
Is this a class somewhere on the page? I can't seem to find where this is located

Reply:
Its the nav bar you and Steve made. I added links to them
<img width="695" alt="Screen Shot 2020-05-30 at 2 34 19 PM" src="https://user-images.githubusercontent.com/7406971/83338548-a96a8500-a282-11ea-8834-3d301e717475.png">